examples/vision/semantic_segmentation_deeplab_v3.py

```diff
@@ -0,0 +1,340 @@
+"""
+Title: Semantic segmentation using DeepLabV3+
+Author: [Alexia Audevart](https://github.com/aaudevart)
+Date created: 2023/13/04
```

Comment:
Is this meant to be September 13? (13/09)?

Either way -- YYYY/MM/DD format is the standard here so let's use that.